crates/astria-bridge-withdrawer/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2025-03-06
```

Comment:
We should add the link for this new version at the foot of this file.  Same for the other changelogs in this PR.